# Induce an exception to see what happens
- ping:
    data: crash
